integration/analyzer_peliasIndexOneEdgeGram.js

```diff
@@ -175,15 +189,23 @@ module.exports.tests.address = function(test, common){
     suite.action( function( done ){ setTimeout( done, 500 ); }); // wait for es to bring some shards up
 
     assertAnalysis( 'address', '101 mapzen place', [
-      '101', 'm', 'ma', 'map', 'mapz', 'mapze', 'mapzen', 'p', 'pl', 'pla', 'plac', 'place'
+      '0:101',
+      '1:m', '1:ma', '1:map', '1:mapz', '1:mapze', '1:mapzen',
+      '2:p', '2:pl', '2:pla', '2:plac', '2:place'
     ]);
 
     assertAnalysis( 'address', '30 w 26 st', [
-      '30', 'w', 'we', 'wes', 'west', '26', 's', 'st', 'str', 'stre', 'stree', 'street'
+      '0:30',
+      '1:w', '1:we', '1:wes', '1:west',
+      '2:26',
+      '3:s', '3:st', '3:str', '3:stre', '3:stree', '3:street'
     ]);
 
     assertAnalysis( 'address', '4B 921 83 st', [
-      '4b', '921', '83', 's', 'st', 'str', 'stre', 'stree', 'street'
+      '0:4b',
+      '2:921', // @todo: this token position is incorrect
+      '3:83',
+      '4:s', '4:st', '4:str', '4:stre', '4:stree', '4:street'
     ]);
```

Comment:
in the process of doing this PR I found one peculiarity, which is a good example of why this is important.

Reply:
woah very strange. any idea how or why this is happening?